source/Plugins/ExpressionParser/Swift/SwiftREPL.h

```diff
@@ -80,6 +80,7 @@ class SwiftREPL : public REPL {
 
 private:
   lldb::SwiftASTContextSP m_swift_ast_sp;
+  bool m_swift_repl_initialized = false;
```

Comment:
This variable name is a bit confusing because it sounds like it keeps track of whether the whole repl is initialized. Maybe something involving the word "completion" would be better. Like `m_completion_module_initialized`.

(I had the same confusion with `g_repl_module_name` and also it's string value "repl" when I initially came across this code, so it might be good to change them too, to something like `static ConstString g_completion_module_name("completion")`.

Reply:
I agree, let me do this.